File: CS/MyBarManager/MyBarManager.cs
using System;
using DevExpress.XtraBars;
using System.ComponentModel;
using DevExpress.XtraBars.Styles;

namespace WindowsApplication1 {
    public class MyBarManager : BarManager {
        public MyBarManager(IContainer container)
            : base(container) {
        }
        public MyBarManager()
            : base() {
            MyBarButtonItem.Register();
        }
    }
}

File: VB/MyBarManager/MyBarManager.vb
Imports DevExpress.XtraBars
Imports System.ComponentModel

Namespace WindowsApplication1

    Public Class MyBarManager
        Inherits BarManager

        Public Sub New(ByVal container As IContainer)
            MyBase.New(container)
        End Sub

        Public Sub New()
            MyBase.New()
            MyBarButtonItem.Register()
        End Sub
    End Class
End Namespace
